### PR TITLE
Fix snapcraft nightly

### PR DESCRIPTION
**Pull request by 5chdn**

### CODE REVIEW

**Comment by 5chdn** [comment]
This is blocked by https://github.com/paritytech/parity-import-tests/issues/3

**Comment by 5chdn** [comment]
Nightly is all green now: https://gitlab.parity.io/parity/parity/pipelines/17901

:tada:

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -142,7 +142,7 @@ linux-snap:
     - rust-stable
   artifacts:
     paths:
-    - scripts/parity_*_amd64.snap
+    - parity.zip
```
Comment (tomusdrw) [inline .gitlab-ci.yml]: Should we add something that indicates that it's a snap? (to don't confuse it with packed binary or sth)

Comment (General-Beck) [reply .gitlab-ci.yml]: it only in artifacts on gitlab. for tests, we are not push in release

**Pull request merged**